### PR TITLE
Removed always empty fields in db api extended account struct

### PR DESCRIPTION
**Pull request by AKorpusenko**

### CODE REVIEW

File: plugins/database_api/include/golos/plugins/database_api/state.hpp [hunk 1]
```diff
@@ -42,22 +42,11 @@ namespace golos {
...
-                vector<pair<string, uint32_t>> tags_usage;
-                vector<pair<account_name_type, uint32_t>> guest_bloggers;
-
                 optional<map<uint32_t, extended_limit_order>> open_orders;
                 optional<vector<string>> comments; /// permlinks for this user
                 optional<vector<string>> blog; /// blog posts for this user
                 optional<vector<string>> feed; /// feed posts for this user
                 optional<vector<string>> recent_replies; /// blog posts for this user
-                map<string, vector<string>> blog_category; /// blog posts for this user
                 optional<vector<string>> recommended; /// posts recommened for this user
```
Comment (afalaleev) [inline plugins/database_api/include/golos/plugins/database_api/state.hpp]: What about fields: open_orders, comments, blog, feed, recent_replies, recommended? Are them used?

File: plugins/database_api/api.cpp [hunk 2]
```diff
@@ -434,12 +420,7 @@ namespace golos {
                     auto itr = idx.find(name);
                     if (itr != idx.end()) {
                         results.push_back(extended_account(*itr, _db));
-                        results.back().reputation = get_account_reputation(name);
                         auto vitr = vidx.lower_bound(boost::make_tuple(itr->id, witness_id_type()));
-                        while (vitr != vidx.end() && vitr->account == itr->id) {
-                            results.back().witness_votes.insert(_db.get(vitr->witness).owner);
```
Comment (afalaleev) [inline plugins/database_api/api.cpp]: Is there any method returned witness_votes?

Comment (AKorpusenko) [reply plugins/database_api/api.cpp]: As I can see, no, there are no any method which returns `witness_votes`

File: plugins/database_api/api.cpp [hunk 3]
```diff
@@ -122,20 +122,6 @@ namespace golos {
 
                 std::vector<account_name_type> get_miner_queue() const;
 
-                share_type get_account_reputation(const account_name_type& account) const {
-                    if (!_follow_api) {
```
Comment (afalaleev) [inline plugins/database_api/api.cpp]: Remove all references to `follow_plugin`:
- include <golos/plugins/follow/plugin.hpp>
- the member `_follow_api` from the `plugin::api_impl`

Comment (AKorpusenko) [reply plugins/database_api/api.cpp]: Done

**Pull request merged**